core/config/constants.js

```diff
@@ -13,6 +13,8 @@
 const DEVTOOLS_RTT_ADJUSTMENT_FACTOR = 3.75;
 const DEVTOOLS_THROUGHPUT_ADJUSTMENT_FACTOR = 0.9;
 
+// Notes: these throttling constants should match what is in Lantern.
```

Comment:
Would it be too much to ask for a test case to validate this? Could just be in `default-config-test.js` or something that checks for strict equality.

Next time we touch these this PR may be long forgotten and this comment unnoticed.